pytermgui/ansi_interface.py

```diff
@@ -221,6 +231,7 @@ def _call_listener(self, event: int, data: Any) -> None:
                 callback(data)
 
     async def _WinSIGWINCH(self, check_again_in: float = 0.5) -> None:
+
```

Comment:
What I meant with the docstring-body thing, is that you would have
```python
def function() -> None:
    """This is a docstring"""

    print("Hello")
```

Reply:
I fixed it